packages/react-charting/src/components/Legends/Legends.base.tsx

```diff
@@ -56,6 +57,19 @@ export class LegendsBase extends React.Component<ILegendsProps, ILegendState> {
     };
   }
 
+  public componentDidMount(): void {
+    const isChartEmpty: boolean = !(this.props.legends && this.props.legends.length);
```

Comment:
Legend is not an individual chart. Don't define an empty state here

Reply:
done